docs/en/serverless/logging/add-logs-service-name.mdx

```diff
@@ -0,0 +1,69 @@
+---
+slug: /serverless/observability/add-logs-service-name
+title: Add a service name to logs
+description: Learn how to add a service name field to your logs.
+tags: [ 'serverless', 'observability', 'overview' ]
+---
+
+Adding the `service.name` field to your logs associates them with the services that generate them.
+You can use this field to view and manage logs for distributed services located on multiple hosts.
+Adding a service name also associates your logs with a service in the <DocLink slug="/serverless/observability/apm-services">service inventory</DocLink>.
+
+To add a service name to your logs either:
+
+- Use the `add_fields` processor through an integration, ((agent)) configuration, or ((filebeat)) configuration.
+- Map an existing field from your data stream to the `service.name` field.
+
+## Use the add fields processor to add a service name
+For log data without a service name, use the [`add_fields` processor](((fleet-guide))/add_fields-processor.html) to add the `service.name` field.
+You can add the processor in an integration's settings or in the ((agent)) or ((filebeat)) configuration.
+
+For example, adding the `add_fields` processor to the inputs section of a standalone ((agent)) or ((filebeat)) configuration would add `your_service_name` as the `service.name` field:
+
+```console
+processors:
+    - add_fields:
+        target: service
+        fields:
+            name: your_service_name
+```
```

Comment:
This is the way the logs onboarding adds the service name, not sure if this is the preferred way but used it for this first draft.